.github/workflows/lint.yml

```diff
@@ -6,10 +6,14 @@ on:
     branches:
       - main
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
```

Comment:
Note: this is the fixed version, per https://github.com/freelawproject/courts-db/pull/108